addons/gamification/views/gamification_goal_views.xml

```diff
@@ -146,56 +146,44 @@
         <field name="model">gamification.goal</field>
         <field name="arch" type="xml">
             <kanban create="false">
-                <field name="definition_id"/>
-                <field name="user_id"/>
-                <field name="current"/>
-                <field name="completeness"/>
                 <field name="state"/>
-                <field name="target_goal"/>
-                <field name="definition_description"/>
                 <field name="definition_condition"/>
                 <field name="definition_suffix"/>
                 <field name="definition_display"/>
-                <field name="start_date"/>
-                <field name="end_date"/>
                 <field name="last_update"/>
                 <templates>
-                    <t t-name="kanban-box">
-                        <div t-attf-class="oe_kanban_card oe_kanban_global_click o_kanban_gamification #{record.end_date.raw_value &lt; record.last_update.raw_value and record.state.raw_value == 'failed' ? 'oe_kanban_color_2' : ''} #{record.end_date.raw_value &lt; record.last_update.raw_value and record.state.raw_value == 'reached' ? 'oe_kanban_color_5' : ''}">
-                            <div class="o_kanban_content text-center">
-                                <p><strong><h4 class="oe_goal_name"><field name="definition_id" /></h4></strong></p>
-                                <img class="o_image_24_cover me-1 rounded" t-att-src="kanban_image('res.users', 'avatar_128', record.user_id.raw_value)" t-att-title="record.user_id.value" t-att-alt="record.user_id.value"/>
+                    <t t-name="kanban-card" class="text-center">
+                        <div t-attf-class="#{record.end_date.raw_value &lt; record.last_update.raw_value and record.state.raw_value == 'failed' ? 'oe_kanban_color_2' : ''} #{record.end_date.raw_value &lt; record.last_update.raw_value and record.state.raw_value == 'reached' ? 'oe_kanban_color_5' : ''}">
```

Comment:
no more classname oe_kanban_color... use the `highlight_color` attribute (set on a computed (not stored) field)

Reply:
Hello @aab-odoo sorry i forgot to put the `TODO` there, i had a doubt on how to put the conditions like this
`<div t-attf-class="#{record.end_date.raw_value &lt; record.last_update.raw_value and record.state.raw_value == 'failed' ? 'oe_kanban_color_2' : ''} #{record.end_date.raw_value &lt; record.last_update.raw_value and record.state.raw_value == 'reached' ? 'oe_kanban_color_5' : ''}">` 
into the `highlight_color` also there isn't any field that i can pass to `highlight_color` which can help me get these kind of colors according to the conditions. Can you kindly give any example on how to achieve it?

Reply:
You need a computed field, which is set to 2 or 5 depending on the condition. Then you simply use that field as highlight_color